examples/step-19/doc/kind

```diff
@@ -0,0 +1 @@
+techniques
```

Comment:
Do you think it might be a good idea to consider adding a "particles" kind?

Reply:
Eventually maybe. The problem is that you end up with too many colors in the tutorial graph, and that things are no longer distinguishable. I'd vote for leaving it as is for now.